apps/editors/helpers.py

```diff
@@ -964,3 +970,8 @@ def queue_tabnav_themes_interactive(context):
 @jinja2.contextfunction
 def is_expired_lock(context, lock):
     return lock.expiry < datetime.datetime.now()
+
+@register.function
+def is_limited_reviewer(request):
+    return GroupUser.objects.filter(group__name='Limited Reviewers',
+                                    user=request.user)
```

Comment:
This is a little fragile because it relies on boolean conversion of the query set. I'd rather see something specific, like:
````
return GroupUser.objects.filter(...).exists()
````
Which also optimizes the query in that memory won't be allocated for the objects resulting from the query.